scripts/gen-env-files.sh

```diff
@@ -46,8 +46,8 @@ POSTGRES_PASSWORD=password
 POSTGRES_DB=geospatial" >> ./database/postgres/.env
 fi
 
-if test -f "./backend/.env"; then
-    echo "./backend/.env exists"
+if test -f "./backend/api/.env"; then
+    echo "./backend/api/.env exists"
```

Comment:
Please since you are changing this file, please update the connection string below. We are using "pims" now as the database name

Reply:
done